tests/functional/search.sh

```diff
@@ -44,3 +44,12 @@ e=$'\x1b' # grep doesn't support \e, \033 or even \x1b
 (( $(nix search -f search.nix foo --exclude 'foo|bar' | grep -Ec 'foo|bar') == 0 ))
 (( $(nix search -f search.nix foo -e foo --exclude bar | grep -Ec 'foo|bar') == 0 ))
 [[ $(nix search -f search.nix -e bar --json | jq -c 'keys') == '["foo","hello"]' ]]
+
+# Tests for flakes
+pushd flakes
+source common.sh
+repoDir="$TEST_ROOT/repo-$RANDOM"
+createGitRepo "$repoDir"
+createSimpleGitFlake "$repoDir"
```

Comment:
To work around the test failure, insert this additional line:

```suggestion
createSimpleGitFlake "$repoDir"
nix flake show "$repoDir" # TODO: Remove when https://github.com/NixOS/nix/issues/9479 was fixed
```